Assets/Scripts/FloatingOrigin/Origin.cs

```diff
@@ -88,19 +88,20 @@ private IEnumerator AttemptShift()
         /// <param name="destination"></param>
         public void MoveOriginTo(Coordinate destination)
         {
-            Coordinate mainShifterRD = new Coordinate(mainShifter.position).Convert(CoordinateSystem.RDNAP);
-           
+            Coordinate CurrentOrigin = new Coordinate(transform.position);
+
 #if UNITY_EDITOR
+            Coordinate mainShifterRD = new Coordinate(transform.position).Convert(CoordinateSystem.RDNAP);
```

Comment:
Ik denk dat deze currentOriginRD moet zijn, gezien dit coordinate het unity 0,0,0 coordinaat van het Coordinate object pakt. Zoiets:

Coordinate currentOriginRD = new Coordinate(transform.position).Convert(CoordinateSystem.RDNAP);

Reply:
heb benamingen in dit blok verbeterd